package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okteto/react-oauth2-login",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
is there a way to handle that via a github release, so we don't have to include it in the commits?

Reply:
Yes, I will create the issue to track it. Not a priority right now since we don't update this repo that often.